File: src/test/java/spoon/test/replace/Foo.java
package spoon.test.replace;

class Foo {
	int i;
	void foo() {
		int x = 3;
		int z;
		z= x+1;
		System.out.println(z);
	}
	
	void bar() {
		int y = 4;
	}
	
}

class Bar {
	float i;
}

File: spoon-kotlin/src/test/kotlin/spoon/test/invocation/testclasses/Foo.kt
package spoon.test.invocation.testclasses

class Foo {
    fun f1() {

    }
    fun f2(i: Int): Bar { return Bar() }

    fun foo() {
        topLevel()
    }

}

class Bar {
    val foo = Foo()
    fun bar() {
        bar()
        foo.f1()
        foo.f2(1)
        foo.f2(2).foo.foo()
    }
}

fun topLevel() {}